api_tests/files/views/test_file_detail.py

```diff
@@ -478,104 +496,114 @@ def test_listing(self):
-            auth=self.user.auth,
+        res = app.get(
+            '/{}files/{}/versions/'.format(API_BASE, file._id),
+            auth=user.auth,
         )
-        assert_equal(res.status_code, 200)
-        assert_equal(len(res.json['data']), 2)
-        assert_equal(res.json['data'][0]['id'], '1')
-        assert_equal(res.json['data'][1]['id'], '2')
-
-    def test_by_id(self):
-        res = self.app.get(
-            '/{}files/{}/versions/1/'.format(API_BASE, self.file._id),
-            auth=self.user.auth,
+        assert res.status_code == 200
+        assert len(res.json['data']) == 2
+        assert res.json['data'][0]['id'] == '1'
+        assert res.json['data'][1]['id'] == '2'
+
+    def test_load_and_property(self, app, user, file):
+        # test_by_id
+        res = app.get(
+            '/{}files/{}/versions/1/'.format(API_BASE, file._id),
+            auth=user.auth,
         )
-        assert_equal(res.status_code, 200)
-        assert_equal(res.json['data']['id'], '1')
+        assert res.status_code == 200
+        assert res.json['data']['id'] == '1'
 
-    def test_read_only(self):
-        assert_equal(self.app.put(
-            '/{}files/{}/versions/1/'.format(API_BASE, self.file._id),
+        # test_read_only
+        assert app.put(
+            '/{}files/{}/versions/1/'.format(API_BASE, file._id),
             expect_errors=True,
-            auth=self.user.auth,
-        ).status_code, 405)
+            auth=user.auth,
+        ).status_code == 405
 
-        assert_equal(self.app.post(
-            '/{}files/{}/versions/1/'.format(API_BASE, self.file._id),
+        assert app.post(
+            '/{}files/{}/versions/1/'.format(API_BASE, file._id),
             expect_errors=True,
-            auth=self.user.auth,
-        ).status_code, 405)
+            auth=user.auth,
+        ).status_code == 405
 
-        assert_equal(self.app.delete(
-            '/{}files/{}/versions/1/'.format(API_BASE, self.file._id),
+        assert app.delete(
+            '/{}files/{}/versions/1/'.format(API_BASE, file._id),
             expect_errors=True,
-            auth=self.user.auth,
-        ).status_code, 405)
-
-
-class TestFileTagging(ApiTestCase):
-    def setUp(self):
-        super(TestFileTagging, self).setUp()
-        self.user = AuthUserFactory()
-        self.node = ProjectFactory(creator=self.user)
-        self.file1 = api_utils.create_test_file(
-            self.node, self.user, filename='file1')
-        self.payload = {
+            auth=user.auth,
+        ).status_code == 405
+
+@pytest.mark.django_db
+class TestFileTagging:
+    @pytest.fixture()
+    def node(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def file1(self, user, node):
+        return api_utils.create_test_file(
+            node, user, filename='file1')
+
+    @pytest.fixture()
+    def payload(self, file1):
+        payload = {
             "data": {
```

Comment:
Replace double quotes (") with single quotes (') wherever possible (basically whenever you don't need to do too much escaping)